sys/Makefile.include

```diff
@@ -31,6 +31,13 @@ ifneq (,$(filter oneway_malloc,$(USEMODULE)))
   USEMODULE_INCLUDES += $(RIOTBASE)/sys/oneway-malloc/include
 endif
 
+ifneq (,$(filter app_metadata,$(USEMODULE)))
+  # Overwrite the interface version
```

Comment:
The comment `# Overwrite the interface version` is still using the very unspecific wording `interface version`. How about something more explicit about the fact that this modifies the reported version of the standardized shell output format? (and to explain the context maybe a reference to the RDM?)